.github/workflows/release-sanitizer-image.yml

```
@@ -0,0 +1,40 @@
+              uses: docker/setup-buildx-action@v1
+        
+            - name: Log in to Docker Hub
+              uses: docker/login-action@v1
+              with:
+                username: ${{ secrets.DOCKER_USERNAME }}
+                password: ${{ secrets.DOCKER_PASSWORD }}
+        
+            - name: Build and push sanitizer image
+              uses: docker/build-push-action@v4
+              with:
+                context: .
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)